template.yaml

```diff
@@ -668,7 +668,7 @@ Resources:
   NvaGetCustomerFunction:
     Type: AWS::Serverless::Function
     Properties:
-      CodeUri: get-customer
+      CodeUri: create-customer
```

Comment:
the common package name will be create-package until I finish merging all customer handlers